### PR TITLE
add label-to-title.yml

### PR DESCRIPTION
**Pull request by samwaseda**

### CODE REVIEW

**Comment by samwaseda** [comment]
The error from [thie page](https://github.com/pyiron/elaston/actions/runs/10382073560/workflow) says:

> invalid value workflow reference: references to workflows must be rooted in '.github/workflows'

So I guess it has to be inside `.github/workflows`.

File: .github/workflows/label-to-title.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+on:
+  pull_request:
+    types: [labeled, unlabeled]
+
+jobs:
+  label-to-title:
+    runs-on: ubuntu-latest
+    uses: pyiron/actions/label-to-title@label-to-title
+    secrets: inherit
```
Comment (liamhuber) [inline .github/workflows/label-to-title.yml]:
```suggestion
    runs-on: ubuntu-latest
    steps:
    - uses: pyiron/actions/label-to-title@label-to-title
```

Comment (liamhuber) [reply .github/workflows/label-to-title.yml]: We might need secrets too at the end of the day, but try this

**Comment by liamhuber** [comment]
` Required property is missing: shell` -- fair, that's just the shell line like the other actions we use bash in. Simple typo.

`Unrecognized named-value: 'secrets'. Located at position 1 within expression: secrets.GITHUB_TOKEN,pyiron/actions/label-to-title/label-to-title/action.yml` -- I guess we need either the `secrets: inherit` line, or there might be something like `github.token` that gets inherited automatically, I need to google.

**Comment by samwaseda** [comment]
Ah it worked. Beautiful

**Comment by liamhuber** [comment]
Also 

```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

You can look at the cached miniforge action for an example of a step holding an output and other steps leveraging it.

**Pull request merged**